src/App.js

```diff
@@ -1,10 +1,39 @@
+// import React from 'react';
+// import { ChakraProvider } from '@chakra-ui/react';
+// import { BrowserRouter, Route, Routes } from 'react-router-dom';
+// import LandingPage from './components/LandingPage';
```

Comment:
You can just remove this commented out code, if we ever want it again we'll go back to github